content/02.delete-me.md

```diff
@@ -51,7 +56,7 @@ two spaces at end of line.
 
 Document section headings:
 
-# Heading 1
+# Heading 1{.center .page_center}
```

Comment:
Is this going to make Heading 1 take an entire page here?

Reply:
Yes.

Reply:
Would it be better to document these options separately from the Heading 1, in case it confuses users.

Reply:
How about something like this, so it suggests that it can be used on any heading:

```
# Heading 1

## Heading 2

### Heading 3

#### Heading 4

### A heading centered on its own printed page{.center .page_center}
```

It can be used on other things too, like paragraphs, but I don't know if we should go nuts showing all the different combinations.

Reply:
> How about something like this, so it suggests that it can be used on any heading:

I like that more!